src/app/app.py

```diff
@@ -10,6 +12,25 @@
 app = FastAPI()
 
 
+@app.exception_handler(500)
+async def internal_exception_handler(request: Request, exc: Exception):
+    """
```

Comment:
How come you're passing 'request' and 'exc' as parameters when they aren't being used in here?